app/views/vacancies/index.html.haml

```diff
@@ -33,15 +33,18 @@
             %ul.govuk-list.govuk-list--bullet
               - t('jobs.no_jobs.suggestions').each do |list_item|
                 %li= list_item
-        - if @vacancies_search.wider_radius_suggestions.present?
+        - if @vacancies_search.wider_search_suggestions.present?
           .divider-bottom
             - if @vacancies_search.keyword.present?
-              .govuk-heading-m= t('jobs.no_jobs.radius_suggestions.heading.keyword_html', keyword: @vacancies_search.keyword)
+              .govuk-heading-m= t('jobs.no_jobs.wider_search_suggestions.heading.keyword_html', keyword: @vacancies_search.keyword)
             - else
-              .govuk-heading-m= t('jobs.no_jobs.radius_suggestions.heading.no_keyword')
+              .govuk-heading-m= t('jobs.no_jobs.wider_search_suggestions.heading.no_keyword')
             %ul.govuk-list.govuk-list--bullet
-              - @vacancies_search.wider_radius_suggestions.each do |wider_radius|
-                %li= govuk_link_to("#{wider_radius.first} miles", jobs_path(@vacancies_search.params_hash.merge(radius: wider_radius.first)), class: "wider-radius-gtm") + ' ' + t('jobs.no_jobs.radius_suggestions.results', count: wider_radius.last)
+              - @vacancies_search.wider_search_suggestions.each do |wider_radius|
+                - if @vacancies_search.location_search.location_polygon.present?
+                  = t('jobs.no_jobs.wider_search_suggestions.suggestion_html', search_link: link_to(t('jobs.no_jobs.wider_search_suggestions.radius_distance', radius: wider_radius.first), jobs_path(@vacancies_search.params_hash.merge(buffer_radius: wider_radius.first))), count: t('jobs.no_jobs.wider_search_suggestions.results', count: wider_radius.last))
```

Comment:
These are quite unreadable, maybe try relative translations or something to make them shorter